main.go

```diff
@@ -123,29 +153,26 @@ func main() {
 	}
 
 	// Create a new manager to provide shared dependencies and start components
-	mgr, err := manager.New(hubCfg, options)
+	mgr, err := ctrl.NewManager(hubCfg, options)
 	if err != nil {
-		log.Error(err, "")
+		log.Error(err, "Failed to start manager")
 		os.Exit(1)
 	}
 
 	log.Info("Registering Components.")
 
-	// Setup Scheme for all resources
-	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
-		log.Error(err, "")
-		os.Exit(1)
-	}
-
 	// Setup all Controllers
-	if err := controller.AddToManager(mgr, managedCfg); err != nil {
-		log.Error(err, "")
+	if err = (&sync.PolicyReconciler{
+		HubClient:       mgr.GetClient(),
+		ManagedClient:   managedClient,
+		ManagedRecorder: managedRecorder,
+		Scheme:          mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		log.Error(err, "unable to create controller", "controller", sync.ControllerName)
 		os.Exit(1)
 	}
```

Comment:
After this line, could you please include the `//+kubebuilder:scaffold:builder` comment?